client/me/help/help-contact/index.jsx

```diff
@@ -578,7 +579,7 @@ const HelpContact = React.createClass( {
 		//    requests are sent to the language specific forums (for popular languages)
 		//    we don't tell the user that support is only offered in English.
 		const showHelpLanguagePrompt =
-			( olark.locale !== i18n.getLocaleSlug() ) &&
+			( currentUserLocale !== i18n.getLocaleSlug() ) &&
```

Comment:
Hm, I put `console.log(currentUserLocale, i18n.getLocaleSlug(), variationSlug);` in here and when I switch my language, `currentUserLocale` and `i18n.getLocaleSlug()` are always the same. E.g. I switch to Afrikaans and the console output is:

```
af af SUPPORT_TICKET
```

[In client boot](https://github.com/Automattic/wp-calypso/blob/master/client/boot/index.js#L144-L158) the locale data gets mapped to locale from `lib/user` — which I imagine is the same source that feeds `currentUserLocale` through Redux?